### DISCUSSION TITLE
Some font files have `NL` in the name. What is this?

### DISCUSSION THREAD

**Discussion by spikespaz** [ryanoasis/nerd-fonts#1517] [Q&A] [answered]
Some font files have `NL` in the name. What is this?

**Comment by JLOrdaz** [comment]
Maybe is "Not Ligature"?

**Reply by Finii** [reply, marked as answer]
This is no Nerd Fonts invention but comes from upstream, the original font source.
They do not explain much:

<img width="1003" alt="Screenshot 2024-06-26 at 23 17 42" src="https://github.com/ryanoasis/nerd-fonts/assets/16012374/f7b504e3-214f-42cf-85c8-691fc8704ee7">

But you can see this on Nerd Fonts:

<img width="1079" alt="Screenshot 2024-06-26 at 23 18 59" src="https://github.com/ryanoasis/nerd-fonts/assets/16012374/55b1eb15-530f-4c6c-aaa5-d3af0f2fd029">

Thanks @JLOrdaz, I did not really notice the question 😬
